prototype/utils/testUtils.js

```diff
@@ -46,6 +51,11 @@ module.exports = {
     }
 
     return RefocusCollectorEval.safeTransform(functionBody, args);
-  },
+  }, // evalTransformFunction
+
+  evalToUrlFunction(functionBody, ctx, aspects, subjects) {
```

Comment:
We needed two steps for evalling the transform because it can be called from doTransform or doHandleError. For toUrl that's not necessary, so generateUrl and evalToUrlFunction should be combined.